src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java

```diff
@@ -198,6 +198,25 @@ private static String getTerminalFieldValue(FieldType field, Terminal terminal)
             case CONNECTED,
                 CONNECTED_1,
                 CONNECTED_2 -> String.valueOf(terminal.isConnected());
+            case REGULATING_TERMINAL ->
+                    terminal != null &&
+                    terminal.getVoltageLevel() != null &&
+                    terminal.getVoltageLevel().getId() != null &&
```

Comment:
A regulating terminal is considered as exists if having voltage level and connectable equipment, in fact it is not necessary to get one of these ids as returned string. Whatever non-empty string is enough since field REGULATING_TERMINAL is typed as string.

I have returned OperatorType.EXISTS.name() as a non-empty string which means a regulating terminal exists. 

DONE in https://github.com/gridsuite/filter/pull/17/commits/dabe9b541c4e65f1f0c95f13c5b6f1b1c06d0242